### ISSUE TITLE
Accessibility swarm on persian/afghanistan IDX page

### ISSUE THREAD

**Issue by DenisHdz** [bbc/simorgh#6408]
**Is your feature request related to a problem? Please describe.**
Carry out a team accessibility swarm for IDX page with all the disciplines once it is completed.

**Describe the solution you'd like**
Team a11y swarm following the criteria and checklist in the spreadsheet (ask @DenisHdz for a link). The main things to check are:

- General page a11y acceptance criteria listed in the second tab on the spreadsheet (skip link, region, H1, etc),
- Key a11y acceptance criteria listed in the second tab on the spreadsheet (live promo etc)
- General AT testing - Quick look over entire page - at one of each type of component/element
- h1: screen reader UX, design, colour contrast, colour blindness (listed in spreadsheet)

ACC (rest in 2nd tab of spreadsheet):
```
Given I'm on the url persian/afghanistan
when I see the page
Then I see the title of the page as a H1 visually displayed

Given I'm on the url persian/afghanistan
when I see the page
Then it is rendered following the same rules as the homepage (except the h1 is visually displayed)
And the content is rendered from https://ares-api.api.bbci.co.uk/api/index/persian/afghanistan

Given I'm on the url persian/afghanistan
when I look at the navigation
Then is it is the same navigation as /persian
and the `Afghanistan` should be highlighted
```

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [X] This feature is expected to need manual testing.

**Additional context**

**Comment by thekp** [comment]
Blocked on: 
https://github.com/bbc/simorgh/issues/6420
https://github.com/bbc/simorgh/issues/6414

**Comment by DenisHdz** [comment]
This can be closed now as no P1 bugs have been found.
